cmd/test/main.go

```diff
@@ -195,6 +211,24 @@ func New(ctx *TestCommandContext) *cobra.Command {
 	return testCommand
 }
 
+func generateTestCommandOptions(testCommandFlags *TestCommandFlags, localConfigContent *localConfig.ConfigContent) *TestCommandOptions {
```

Comment:
I think this can me part of the TestCommandOptions struct. I'm not sure thought if we want it to be public.

Reply:
I think I'll keep it here for now